### ISSUE TITLE
Bug: Assets loader crash on drawImage 

### ISSUE THREAD

**Issue by rscheumann** [avocode/avocode#430]
The crash happened while opening a previously opened PSD file from the project manager.

-----
App v.2.4.2-d618096
Source UID: 75646
```
TypeError: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The provided value is not of type '(HTMLImageElement or HTMLVideoElement or HTMLCanvasElement or ImageBitmap)'
    at TypeError (native)
    at t.module.exports.t._loadMaskedBy (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:3710)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:1011
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:3245
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:200:33
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:3183
    at replenish (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:191:21)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:203:33
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:3183
    at replenish (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:191:21)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:203:33
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:3183
    at replenish (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:191:21)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:203:33
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/assets-loader.js:1:3183
    at r
```

**Comment by petrbrzek** [comment]
Thanks for additional information. We'll dig into it.

**Comment by rychlis** [comment]
This should be fixed since around 2.4.3, I am closing the issue
